src/concrete/ml/sklearn/base.py

```diff
@@ -1116,9 +1115,11 @@ def quantize_input(self, X: numpy.ndarray) -> numpy.ndarray:
         assert isinstance(q_X, numpy.ndarray)
         return q_X
 
-    def dequantize_output(self, q_y_preds: numpy.ndarray) -> numpy.ndarray:
+    def dequantize_output(self, *q_y_preds: numpy.ndarray) -> numpy.ndarray:
```

Comment:
just realized here, we have a signature change here : `dequantize_output` is a common method between all built-in models, this means it 'inherits' from the BaseEstimator method, which has the following signature / docstring : 
```
    @abstractmethod
    def dequantize_output(self, q_y_preds: numpy.ndarray) -> numpy.ndarray:
        """De-quantize the output.

        This step ensures that the fit method has been called.

        Args:
            q_y_preds (numpy.ndarray): The quantized output values to de-quantize.

        Returns:
            numpy.ndarray: The de-quantized output values.
        """
```

what should we do then here ? I feel like this signature change is necessary (I guess QNNs could be multi-output right ?) but `dequantize_output` for linear / tree models should stay like it is right now 

I'm actually surprised mypy did not complain here, I thought it would point out that the method's signature has changed .. ? 

so yes, not sure what could be done here, it probably looks fine to just keep it liks this but maybe add a new docstring to better indicate that the signature is a bit different ?